src/test/java/com/artipie/RepoPermissionsFromSettingsTest.java

```diff
@@ -103,6 +110,88 @@ void returnsEmptyMapWhenPermissionsAreNotSet() {
         );
     }
 
+    @Test
+    void updatesUserPermissions() throws IOException {
```

Comment:
@olenagerasimova I think we miss the case when user has some permissions and we update the user by adding more permissions or removing some permissions